pkgs/servers/sql/mssql/jdbc/jtds.nix

```diff
@@ -0,0 +1,27 @@
+{lib, stdenv, fetchurl, unzip}:
+
+stdenv.mkDerivation rec {
+  pname = "jtds";
+  version = "1.3.1";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/jtds/files/jtds/${version}/${pname}-${version}-dist.zip";
```

Comment:
Please use a mirror:// style URL.

Reply:
Thanks for the review, I've changed my pull request to use a mirror URL.